python/py-absl/Portfile

```diff
@@ -18,16 +18,15 @@ homepage            https://github.com/abseil/abseil-py
 master_sites        pypi:a/absl-py
 distname            absl-py-${version}
 
-checksums           rmd160  e2fa96fb03bedd2a4d5e4de09969bf116ab7f593 \
-                    sha256  d9129186431e150d7fe455f1cb1ecbb92bb5dba9da9bc3ef7b012d98c4db2526 \
-                    size    103181
+checksums           rmd160  64942f2658e1951133fe14ab388f375399f23d7f \
+                    sha256  b20f504a7871a580be5268a18fbad48af4203df5d33dbc9272426cb806245a45 \
+                    size    110054
 
-python.versions     27 35 36 37 38
+python.versions     37 38
```

Comment:
you cannot remove subports on which other ports still depend:
```
>>> port echo depends:py27-absl ; port echo depends:py35-absl ; port echo depends:py36-absl
py27-tensorboard1
py27-tensorflow1
py36-tensorboard1
py36-tensorflow1
```

Reply:
also when restoring the PY27 subport, please note that the `py-enum34` dependency is missing so please add that below as:
```
if {${python.version} == 27} {
    depends_lib-append port:py${python.version}-enum34
}
```

Reply:
Thanks done.